functions-methods.js

```diff
@@ -20,7 +23,27 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
-
+function typeOfEmail (emailAddress) {
+    const domain = emailAddress.substring(emailAddress.indexOf('@') + 1);
+    let typeOfEmail = 'Extern';
+    switch(domain) {
+        case 'novi.nl':
+            typeOfEmail = 'Medewerker'; 
+            break;
+        case 'novi-education.nl':
+            typeOfEmail = 'Student'
+            break;
+        default:
+            break;
+    }
+    return typeOfEmail;
```

Comment:
Als je na het switch-statement niets meer met de waarde gaat doen, dan kun je deze ook direct teruggeven:
```suggestion
    switch(domain) {
        case 'novi.nl':
            return 'Medewerker';
        case 'novi-education.nl':
            return 'Student';
        default:
            return 'Extern';
    }
```